Jenkinsfile

```diff
@@ -115,18 +129,42 @@ pipeline {
             expression { return params.Run_As_Master_Branch }
             triggeredBy 'TimerTrigger'
             changeRequest()
+            expression { return env.TAV_UPDATED != "false" }
           }
           expression { return params.tav_ci }
         }
       }
       steps {
-        withGithubNotify(context: 'TAV Test', tab: 'tests') {
-          deleteDir()
-          unstash 'source'
-          dir("${BASE_DIR}"){
-            script {
-              def node = readYaml(file: '.ci/.jenkins_tav_nodejs.yml')
-              def tav = readYaml(file: '.ci/.jenkins_tav.yml')
+        deleteDir()
+        unstash 'source'
+        dir("${BASE_DIR}"){
+          script {
+            def ghContextName = 'TAV Test'
```

Comment:
I'd extract this to a function to give context about what it does.

Reply:
See https://github.com/elastic/apm-agent-nodejs/pull/1228/commits/adb4b08e14059269931f4f9c6806a34ef7f7597d